src/cuda/kernels/distributed.f90

```diff
@@ -1,653 +1,653 @@
-module m_cuda_kernels_dist
-   use cudafor
+  module m_cuda_kernels_dist
+    use cudafor
 
-   use m_common, only: dp
+    use m_common, only: dp
 
-   implicit none
+    implicit none
 
-contains
+  contains
 
-   attributes(global) subroutine der_univ_dist( &
+    attributes(global) subroutine der_univ_dist( &
       du, send_u_s, send_u_e, u, u_s, u_e, coeffs_s, coeffs_e, coeffs, n, &
       ffr, fbc, faf &
       )
-      implicit none
-
-      ! Arguments
-      real(dp), device, intent(out), dimension(:, :, :) :: du, send_u_s, &
-                                                           send_u_e
-      real(dp), device, intent(in), dimension(:, :, :) :: u, u_s, u_e
-      real(dp), device, intent(in), dimension(:, :) :: coeffs_s, coeffs_e
-      real(dp), device, intent(in), dimension(:) :: coeffs
-      integer, value, intent(in) :: n
-      real(dp), device, intent(in), dimension(:) :: ffr, fbc, faf
-
-      ! Local variables
-      integer :: i, j, b, k, lj
-      integer :: jm2, jm1, jp1, jp2
-
-      real(dp) :: c_m4, c_m3, c_m2, c_m1, c_j, c_p1, c_p2, c_p3, c_p4, &
-                  temp_du, alpha, last_r
-
-      i = threadIdx%x
-      b = blockIdx%x
-
-      ! store bulk coeffs in the registers
-      c_m4 = coeffs(1); c_m3 = coeffs(2); c_m2 = coeffs(3); c_m1 = coeffs(4)
-      c_j = coeffs(5)
-      c_p1 = coeffs(6); c_p2 = coeffs(7); c_p3 = coeffs(8); c_p4 = coeffs(9)
-      last_r = ffr(1)
-
-      du(i, 1, b) = coeffs_s(1, 1)*u_s(i, 1, b) &
-                    + coeffs_s(2, 1)*u_s(i, 2, b) &
```

Comment:
I think this would be very confusing, very surprised fprettify allows this to be honest.

Reply:
Again not visible here on the comment tab, but `+ coeff...` starts from all the way left with two spaces, however I would expect it to start from right below the previous variable after the `=`

Reply:
yeah, I didn't see any way in the manual to indent continuation lines after a `=` nor a `::` with findent

Reply:
with `::` its not the end of the world, one can argue its better to follow findent way, but with `+` I think things would get very confusing and hard to follow.